### DISCUSSION TITLE
Official demo booster-ui won't load on windows

### DISCUSSION THREAD

**Discussion by Superskyyy** [apache/skywalking#11827] [Q&A] [answered]
Tried browsers with cleared cache etc. still, the UI doesn't load. This happens from time to time but I was never really sure what happened.

1. Microsoft edge
2. Chrome 

Symptom is captured in the following screenshot: (i.e., no console errors, all resources seem to load properly)
![image](https://github.com/apache/skywalking/assets/26076517/a4e704aa-d30b-46fd-a56d-261ae3907cb6)

P.S. The demo loads successfully for me on my iphone safari under the same network.

**Comment by wu-sheng** [comment]
Errrr, we don't have Windows for most of the team. FYI @Fine0830

**Reply by wu-sheng** [reply]
BTW, it works on my Mac Edge browser.

<img width="1917" alt="image" src="https://github.com/apache/skywalking/assets/5441976/86a642e1-1fd8-4ea3-9c01-c418f8385ddc">

**Reply by Superskyyy** [reply]
This somehow happen only on the demo site, never saw it in a local deployment though? I'm not even sure its due to deployment issue or my network or the UI itself. Tried to access with/without vpn, but since my phone works i suppose its not the network.

**Reply by wankai123** [reply]
It works on my win10 machine :)
![image](https://github.com/apache/skywalking/assets/16773043/09f1a016-3d16-4d75-92d5-51534b502e46)

**Comment by Superskyyy** [comment, marked as answer]
oh crap, nvm! I know the reason, its due to the path. My browser somehow pointed to /general but i think the path is gone in the newer version.
